include/mbedtls/config_adjust_legacy_from_psa.h

```diff
@@ -440,6 +440,41 @@
 
 /* End of ECC section */
 
+#if defined(PSA_WANT_DH_RFC7919_2048)
+#if !defined(MBEDTLS_PSA_ACCEL_DH_RFC7919_2048)
+#define MBEDTLS_PSA_BUILTIN_DH_RFC7919_2048 1
+#define MBEDTLS_DHM_C
```

Comment:
I don't think `DHM_C` is what we want here: the built-in implementation of FFDH in PSA only relies on `BIGNUM_C` IIRC.

Also, I'd prefer using a temporary macro, like `#define MBEDTLS_PSA_DH_ACCEL_INCOMPLETE_PARAMS` here (and for each size) and then on line 479 change `#if !defined(MBEDTLS_PSA_ACCEL_ALG_FFDH)` by appending `|| defined(MBEDTLS_PSA_DH_ACCEL_INCOMPLETE_PARAMS)` - this centralizes enabling `BIGNUM_C` in one place.

Reply:
> I don't think `DHM_C` is what we want here: the built-in implementation of FFDH in PSA only relies on `BIGNUM_C` IIRC.
> 
> Also, I'd prefer using a temporary macro, like `#define MBEDTLS_PSA_DH_ACCEL_INCOMPLETE_PARAMS` here (and for each size) and then on line 479 change `#if !defined(MBEDTLS_PSA_ACCEL_ALG_FFDH)` by appending `|| defined(MBEDTLS_PSA_DH_ACCEL_INCOMPLETE_PARAMS)` - this centralizes enabling `BIGNUM_C` in one place.

Right, sorry. I probably get confused by seeing definitions of binary data as `MBEDTLS_DHM_RFC7919_FFDHExxxx_P_BIN`, but you are right: only BIGNUM functions are used here.